src/main/java/edu/ie3/util/quantities/QuantityUtil.java

```diff
@@ -52,6 +75,24 @@ private QuantityUtil() {
     return Math.abs(aVal - bVal) <= absQuantityTolerance;
   }
 
+  /**
+   * Compares two {@link Quantity}s, if they are considerably equal. This is foremost important for
```

Comment:
Please adapt JavaDoc to highlight the concept of equivalence in contrast to equality.